packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3687,8 +3687,8 @@ export class ContainerRuntime
 				this.submit(message);
 				break;
 			case ContainerMessageType.GC:
-				// GC op is only sent in summarizer which should never reconnect.
-				throw new LoggingError("GC op not expected to be resubmitted in summarizer");
+				this.submit(message);
```

Comment:
@agarwal-navin - Is it worth adding logic here or in a function in GC to check that the subtype is resubmit-able?  (`Delete` is not, `TombstoneLoaded` is)

Reply:
Maybe just asserting that this is not the summarizer client is better.